src/main/java/com/hyperwallet/clientsdk/model/HyperwalletUser.java

```diff
@@ -24,7 +24,7 @@
 
     public static enum Status {PRE_ACTIVATED, ACTIVATED, LOCKED, FROZEN, DE_ACTIVATED}
 
-    public static enum VerificationStatus {UNDER_REVIEW, VERIFIED, REQUIRED, NOT_REQUIRED}
+    public static enum VerificationStatus {UNDER_REVIEW, VERIFIED, REQUIRED, NOT_REQUIRED, REQUESTED}
```

Comment:
Please add the missing statuses EXPIRED, READY_FOR_REVIEW and FAILED, run the tests again.

Reply:
added missing statuses and run the test cases working fine . Thanks @Rrathinasabapath